keyboards/kbdfans/kbd75_round2/rules.mk

```
@@ -0,0 +1,22 @@
+# MCU name
+MCU = atmega32a
+
+# Bootloader selection
+#   Teensy       halfkay
+#   Pro Micro    caterina
+#   Atmel DFU    atmel-dfu
+#   LUFA DFU     lufa-dfu
+#   QMK DFU      qmk-dfu
+#   ATmega32A    bootloadHID
+#   ATmega328P   USBasp
+BOOTLOADER = bootloadHID
+
+# build options
+BOOTMAGIC_ENABLE = full
```

We prefer [Bootmagic Lite](https://docs.qmk.fm/#/feature_bootmagic?id=bootmagic-lite) for any new keyboards, as it eliminates a major source of confusion around key swaps, disabled keys, eeprom reset, etc.
```suggestion
BOOTMAGIC_ENABLE = lite
```